processes/third_party_vuln_process.md

```diff
@@ -0,0 +1,86 @@
+This document describes the management of vulnerabilities within the Node.js
+ecosystem. Vulnerabilities in Node.js core are out of this scope.
+
+# Definitions
+
+* package: in this document, a package is a module available widely to use 
+with Node.js. Most modules are hosted on the npmjs.org repository. 
+
+# Process
+
+Individuals who find potential vulnerabilities in a package are invited 
+to complete a vulnerability report on the dedicated HackerOne organization:
+
+[https://hackerone.com/nodesecurity](https://hackerone.com/nodesecurity)
+
+When a potential vulnerability is reported, the following actions are taken:
+
+## Triage
+
+**Who:** The triage team
```

Comment:
should this list who the members of the triage team are? I assume this is made up of the current recipients of the `security@` messages?

Reply:
Or is this specifically only for module vulnerabilities?

Reply:
@jasnell This is a separate group for triage of non-core vulnerabilities.

Reply:
once #56 lands, this PR can also include addition of a Team to the doc describing privacy policy, because the same privacy policy should apply to thirdparty vulns as to core (I think)